plugin/src/main/java/org/wildfly/plugin/provision/PackageServerMojo.java

```diff
@@ -305,9 +342,45 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             getLog().debug(String.format("Skipping " + getGoal() + " of %s:%s", project.getGroupId(), project.getArtifactId()));
             return;
         }
+        deploymentResolution = DeploymentResolution.getInstance(project);
         super.execute();
     }
 
+    private void deploy(Path deploymentContent, String targetName) throws MojoDeploymentException {
+        if (Files.exists(deploymentContent)) {
```

Comment:
Shouldn't we be doing something like throwing an error if the file does not exist? Or at least logging a warning/error.

Reply:
Good catch, logging a warning.